### PR TITLE
Issue/2

### PR DESCRIPTION
**Pull request by asiday**

### CODE REVIEW

File: crawler/pom.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>dev.asida</groupId>
+		<artifactId>news-aggregator</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+	<groupId>dev.asida</groupId>
```
Comment (yambay) [inline crawler/pom.xml:10]: Remove the group id as it is inherited from the parent

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-parent</artifactId>
+      <version>2.6.4</version>
+      <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <groupId>dev.asida</groupId>
```
Comment (yambay) [inline pom.xml:15]: Remove group id as it is inherited from the parent

File: service/pom.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+      <groupId>dev.asida</groupId>
+      <artifactId>news-aggregator</artifactId>
+      <version>1.0-SNAPSHOT</version>
+    </parent>
+	<groupId>dev.asida</groupId>
```
Comment (yambay) [inline service/pom.xml:10]: Remove group id as it is inherited from the parent

File: crawler/pom.xml (hunk 1 above)
Comment (asiday) [reply crawler/pom.xml:10]: Done

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+
```
[the remark this answers is not in the document]
Comment (asiday) [reply pom.xml:5]: Done

File: service/pom.xml (hunk 3 above)
Comment (asiday) [reply service/pom.xml:10]: Done

**Pull request merged**